plugins/orchestrator-common/src/constants.ts

```diff
@@ -62,7 +62,7 @@ export const workflow_yaml_sample: WorkflowSample = {
 
 // Default values for the orchestrator plugin configuration
 export const default_sonataflow_container_image =
-  'quay.io/kiegroup/kogito-swf-devmode-nightly:main-2023-08-30';
+  'quay.io/kiegroup/kogito-swf-devmode-nightly:main-2023-12-12';
```

Comment:
TODO: Update the tag after https://github.com/apache/incubator-kie-kogito-runtimes/pull/3328.